### ISSUE TITLE
BatchNorm

### ISSUE THREAD

**Issue by hendriks73** [HarisIqbal88/PlotNeuralNet#33]
Nice project! A batch normalization element would be cool...

**Comment by soylisK** [comment]
> Nice project! A batch normalization element would be cool...

Keep in mind that you can copy an all ready layer, like to_Conv() but with different color and actually create your own layer-box. And by adding your caption, It does the job!
